### DISCUSSION TITLE
vlc player

### DISCUSSION THREAD

**Discussion by uzuki7777** [microsoft/winget-pkgs#89237] [General]
Hello, I think this is the right place to alert about this.  winget seems to be getting a nightly version of vlc player for some reason.

**Comment by uzuki7777** [comment]
Right I have regular vlcplayer not nightly but it wants me to download
nightly when I say winget upgrade.

**Comment by uzuki7777** [comment]
Awesome did somebody fix it? It doesn't want me to download vlcplayer anymore!

**Comment by uzuki7777** [comment]
Hello, so it was right that there was another vlc player available, but after I installed it, it still thinks I have the older version. I checked regedit and it had the correct value on it. I saw somewhere I need to fix that, but that didn't need fixing this time. I even rebooted.

**Reply by mdanish-kh** [reply]
Can you please share the output of `winget list VideoLAN.VLC`?

**Reply by uzuki7777** [reply]
![output](https://user-images.githubusercontent.com/118627425/205636900-f1196130-c501-4a2d-a21b-49240e35095f.PNG)

**Reply by mdanish-kh** [reply]
I don't see anything wrong with the output. You are on a previous version and will get the latest if you run `winget upgrade VideoLAN.VLC`. What version are you referring to when you say
> it still thinks I have the older version. I checked regedit and it had the correct value on it

Can you please check the version of VLC you have in `appwiz.cpl` (Control Panel)?

**Comment by uzuki7777** [comment]
If I go to ABOUT on my vlc, it says I have the lateast version. When I go
home I'll try it again and if it's still wrong I'll show you.

**Reply by uzuki7777** [reply]
![vlc](https://user-images.githubusercontent.com/118627425/205770952-2b76bbfd-7f9f-46ed-8c37-7c616ff0d031.PNG)
Yeah it's wrong I have the newer version.

**Comment by nascentt** [comment]
ive been having this exact same issue,  stable being updated to nightly, and the nightly crashes every time i use it, so it's infuriating.  if i uninstall vlc nightly and then install the stable everything works, but the next time i do winget upgrade --all it'll update vlc to nightly and it is essentially unusable again.

(neither https://github.com/microsoft/winget-pkgs/pull/89240 or https://github.com/microsoft/winget-pkgs/pull/89239 seem to have solved it)

**Comment by uzuki7777** [comment]
Well I just use winget upgrade to list updates and then have the program
update itself.
